/*
 * Copyright 2015 Twitter, Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *   http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

#ifndef SRC_CPP_PROTO_MESSAGES_H_
#define SRC_CPP_PROTO_MESSAGES_H_

#include "proto/common.pb.h"
#include "proto/stats.pb.h"
#include "proto/topology.pb.h"
#include "proto/execution_state.pb.h"
#include "proto/physical_plan.pb.h"
#include "proto/tuple.pb.h"
#include "proto/tmaster.pb.h"
#include "proto/scheduler.pb.h"
#include "proto/stmgr.pb.h"
#include "proto/metrics.pb.h"

#endif  // SRC_CPP_PROTO_MESSAGES_H_
